btr-web/btr-layouts/components/footer/BcrosFooter.vue

```diff
@@ -29,17 +19,22 @@
   </footer>
 </template>
 
-<script lang="ts">
-import pkg from '../../package.json'
-
-export default {
-  setup () {
-    const appVersion = pkg.version
-    return {
-      appVersion
-    }
+<script setup lang="ts">
+const { appVersion } = defineProps({
```

Comment:
For this case you can simply do this:
`defineProps<{appVersion: string}>()`
It's a required value so you don't need a default and you can access props directly in the dom